packages/eslint-plugin-svelte/tests/fixtures/rules/no-unused-svelte-ignore/valid/svelte-ignore-comma-separated-input.svelte

```diff
@@ -0,0 +1,5 @@
+<!-- svelte-ignore a11y_label_has_associated_control, a11y_no_noninteractive_tabindex -->
```

Comment:
To exclude Svelte v4/v3 tests, add a `svelte-ignore-comma-separated-requirements.json` file and write the following:

```json
{
	"svelte": ">=5.0.0"
}
```

Reply:
By the way, the reason for the failure is that the error message format is different in old Svelte 😅

Reply:
i did add a requirements file, the one above this comment

but maybe i put it in the wrong place or something 🤔